manifests/n/Nushell/Nushell/0.76.0/Nushell.Nushell.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with WinGet Releaser v2 using Komac v1.0.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Nushell.Nushell
+PackageVersion: 0.76.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- nu
+ReleaseDate: 2023-02-21
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/nushell/nushell/releases/download/0.76.0/nu-0.76.0-x86_64-pc-windows-msvc.msi
+  InstallerSha256: B010A33F982CA2685677122CAD651101F8C93016F59F07F54C88AE2024DA9C44
+  ProductCode: '{6B693F7E-7721-4816-A4ED-12A651A10378}'

```

```suggestion
  ProductCode: '{8EE1FEE1-80B8-47AC-99B4-39D1EED33CBE}'
```

Match new `ProductCode` as well